.eslintrc

```diff
@@ -13,7 +13,7 @@ rules:
   comma-spacing: [2, {before: false, after: true}]
   comma-style: [2, "last"]
   handle-callback-err: [2, "^.*(e|E)rr" ]
-  indent: ["error", 2, {"SwitchCase": 1}]
+  indent: ["error", 2]
```

Comment:
Hi @mattrichard It seems that the `indent` is not overridden by the "prettier', would you please take a look if it's configured correctly? (currently, `npm run lint` has errors, pls see the CI output)